tests/plugins/plugin.chapters.test.js

```diff
@@ -51,3 +100,38 @@ describe('Plugin: Menu Toggle', () => {
     expect($.ajax).toHaveBeenCalled();
   });
 });
+
+describe('updateTOCState', () => {
+
+  beforeEach(() => {
+    window.HTMLElement.prototype.scrollIntoView = function() {};
+    br.getPageIndex = (str) => parseInt(str);
+    br.init();
+    br.updateTOC(SAMPLE_TOC);
```

Comment:
Should this be here?
```suggestion
```

Reply:
Thought it was needed to initialize table of contents, but tests work without it as well